src/cmd/cmd.zig

```diff
@@ -651,19 +649,11 @@ pub fn run() !void {
 
 /// entrypoint to print (and create if NONE) pubkey in ~/.sig/identity.key
 fn identity() !void {
-    var std_logger = try ChannelPrintLogger.init(.{
```

Comment:
do we want to add the RPC server to any command?

Reply:
I think I'll rather do this in a follow-up PR, to keep the surface area of this PR small, and so that it can be added in after fixing the snapshot manifest loading/generation (meaning it will actually be reachable and testable).

Reply:
makes sense - can you add a ticket + link in the comments? then can resolve

Reply:
https://github.com/orgs/Syndica/projects/2/views/10?pane=issue&itemId=89375974